lighthouse-core/test/chromium-web-tests/run-web-tests.sh

```diff
@@ -6,21 +6,11 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 ##
 
-set -eo pipefail
+set -euo pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT="$SCRIPT_DIR/../../.."
 
-if [ x"$BLINK_TOOLS_PATH" == x ]; then
-  echo "Error: Environment variable BLINK_TOOLS_PATH not set"
-  exit 1
-fi
-
-if [ x"$DEVTOOLS_PATH" == x ]; then
-  echo "Error: Environment variable DEVTOOLS_PATH not set"
-  exit 1
-fi
-
 unset -v latest_content_shell
 for file in "$LH_ROOT/.tmp/chromium-web-tests/content-shells"/*/; do
```

Comment:
`-u` breaks the pattern.

Reply:
gotcha. thanks for pointing it out.

took me a bit to understand what this was doing... so.. 

i rewrote it. I think it's a clearer now, plus no issue with unsettedness. :D https://github.com/GoogleChrome/lighthouse/pull/11331/commits/0b134d249cfe10b95eac9b165d6841775d448d7f
ptal?